### DISCUSSION TITLE
Make comments searchable in posting and transactions.

### DISCUSSION THREAD

**Discussion by avinashkanaujiya** [ananthakumaran/paisa#103] [Ideas]
What title said...

**Comment by ananthakumaran** [comment]
available on master https://github.com/ananthakumaran/paisa/commit/cb6bba55d4e2d01ddd5775acb3ecc31c899f721d

**Reply by avinashkanaujiya** [reply]
```
;  TODO
```
But couldn't search the transaction with `TODO` in transactions or postings page.
Am I doing something wrong?

**Reply by ananthakumaran** [reply]
Can you show the full transaction? in the example below, Global is at the transaction level, Posting is at the posting level. If you place it outside the transaction it may not get associated.

```
2022/01/31 Interest
    ; Global note
    ; :interest:
    Income:Interest:Checking                  -1,000 INR ; Posting note
    Assets:Checking
```

**Reply by avinashkanaujiya** [reply]
```
2022/01/31 Interest
    ; TODO
    Income:Interest:Checking                  -1,000 INR
    Assets:Checking
```
I did this, but unable to search in transactions by writing `TODO` in search bar in transactions page.

**Reply by ananthakumaran** [reply]
did you try `note = TODO`?, check the docs for sample https://paisa.fyi/reference/bulk-edit/#property

**Reply by avinashkanaujiya** [reply]
I thought it is full-text search enabled, my bad.
